inspire/modules/deposit/forms.py

```diff
@@ -463,24 +463,28 @@ class LiteratureForm(WebDepositForm):
                                              mapper=journal_title_kb_mapper)
     )
 
-    page_range_article_id = fields.TextField(
+    journal_page = fields.TextField(
         label=_('Page Range/Article ID'),
         description=_('e.g. 1-100'),
+        export_key='page_range_article_id',
         widget_classes="form-control" + ARTICLE_CLASS
```

Comment:
It's just so they start with the same name and then I map them more easily [here](https://github.com/pedrogaudencio/inspire-next/blob/arxiv-split-pub-note/inspire/modules/deposit/views.py#L47). I can change the `export_key` to `page_artid` since it makes sense. ;)